_api-reference/index-apis/recover.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: default
+title: Recovery
+parent: Index APIs
+nav_order: 40
+---
+
+# Recovery API
+Introduced 1.0
+{: .label .label-purple }
+
+The Recovery API provides information about any completed or ongoing shard recoveries for one or more indexes. If a data stream is listed, the API returns information about that data stream's backing indexes. 
+
+Shard recovery involves creating a shard copy to restore a primary shard from a snapshot or to synchronize a replica shard. After the shard recovery process completes, the recovered shard becomes available for use in search and index operations.
+
+Shard recovery occurs automatically in the following scenarios:
+
+- Node startup, known as a local store recovery.
+- Replication of a primary shard.
+- Relocation of a shard to a different node within the same cluster.
+- Restoration of a snapshot.
+- Clone, shrink, or split operations.
+
+The Recovery API reports solely on completed recoveries for shard copies presently in the cluster. It reports only the most recent recovery for each shard copy and does not include historical information about previous recoveries or information about recoveries of shard copies that no longer exist. Consequently, if a shard copy completes a recovery and is subsequently relocated to a different node, the information about the original recovery is not displayed in the Recovery API.
+
+
+## Path and HTTP methods
+
+```
```

Comment:
```suggestion
```json
```

Reply:
@Naarcha-AWS I commented on [this PR](https://github.com/opensearch-project/documentation-website/pull/7641) about this as well. Based on the [API template](https://github.com/opensearch-project/documentation-website/blob/main/templates/API_TEMPLATE.md), shouldn't this have highlighting?